### PR TITLE
Update publishing-to-app-store.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

**Comment by icarusdust** [comment]
Fixes #299

File: content/flutter-publishing/publishing-to-app-store.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ It is also worth pointing out the necessity for each uploaded binary to have a *
 
 {{<notebox>}}Please note that you will need to create an **app record** in App Store Connect before you can automate publishing with Codemagic. It is recommended to upload the very first version of the app manually. {{</notebox>}}
 
+In order to be able to test iOS apps on Apple devices, manual device UDID registration in the Apple Developer account is required or manual registration can be replaced by Codemagic automatic device registration. More information can be found [here](https://docs.codemagic.io/testing/ios-provisioning/). 
+
```
Comment (nevercode-kevin) [inline content/flutter-publishing/publishing-to-app-store.md:25]:
```suggestion
In order to be able to test iOS apps on Apple devices, manual device UDID registration in the Apple Developer Program account is required. Alternatively, Codemagic's automatic device registration can be used to register devices as explained [here](https://docs.codemagic.io/testing/ios-provisioning/). 
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Changes made!

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**